### DISCUSSION TITLE
API被OpenAI关了。

### DISCUSSION THREAD

**Discussion by Panlichen** [nextai-translator/nextai-translator#195] [Q&A]
附上OpenAi的邮件：
Hi there,

After a thorough investigation, we have determined that you or a member of your organization are using the OpenAI API in ways that violate our policies.

Due to this breach we are halting access to the API immediately for the organization Personal. Common reasons for breach include violations of our [usage policies](https://mandrillapp.com/track/click/31165340/openai.com?p=eyJzIjoiUmF2TnhWU2d5dDZoT09lOHdrRG9ZQWpHWHNZIiwidiI6MSwicCI6IntcInVcIjozMTE2NTM0MCxcInZcIjoxLFwidXJsXCI6XCJodHRwczpcXFwvXFxcL29wZW5haS5jb21cXFwvcG9saWNpZXNcXFwvdXNhZ2UtcG9saWNpZXNcIixcImlkXCI6XCJlZTZlMjhkNDRlOGQ0Yzk3YWRkMTliMTg1NmM1OTAzY1wiLFwidXJsX2lkc1wiOltcIjI3YmVlMTJkYjA1YTgyNmQ4MjZjYWE4YmUzOWJkNjZmZjJkZDIzM2VcIl19In0) or accessing the API from an [unsupported location](https://mandrillapp.com/track/click/31165340/platform.openai.com?p=eyJzIjoiZHNhRk9vZFNnSnhqZW0tNVdMQ1lyRVBPdGg0IiwidiI6MSwicCI6IntcInVcIjozMTE2NTM0MCxcInZcIjoxLFwidXJsXCI6XCJodHRwczpcXFwvXFxcL3BsYXRmb3JtLm9wZW5haS5jb21cXFwvZG9jc1xcXC9zdXBwb3J0ZWQtY291bnRyaWVzXCIsXCJpZFwiOlwiZWU2ZTI4ZDQ0ZThkNGM5N2FkZDE5YjE4NTZjNTkwM2NcIixcInVybF9pZHNcIjpbXCJjNWRlYzFkNzY0NDYzNDcwZjBiZmVhYTVjN2ZhMTdhM2FlNDkwMzNjXCJdfSJ9). You may also wish to review our [Terms of Use](https://mandrillapp.com/track/click/31165340/openai.com?p=eyJzIjoiYXpmX252Wm1BaXhtYVR2dkhnWk1QcGlTTkZzIiwidiI6MSwicCI6IntcInVcIjozMTE2NTM0MCxcInZcIjoxLFwidXJsXCI6XCJodHRwczpcXFwvXFxcL29wZW5haS5jb21cXFwvcG9saWNpZXNcXFwvdGVybXMtb2YtdXNlXCIsXCJpZFwiOlwiZWU2ZTI4ZDQ0ZThkNGM5N2FkZDE5YjE4NTZjNTkwM2NcIixcInVybF9pZHNcIjpbXCIzZWE0NGM1NDNkYmY4YmRkMDQ4MmE2N2UzZTBkZWQ3NDFmNWQ5MTJmXCJdfSJ9).

If you believe this is in error and would like to appeal, please contact us through our [help center](https://mandrillapp.com/track/click/31165340/help.openai.com?p=eyJzIjoiNk9OeWVlMHdyOVJQR29LWmkySUQ5Y0wwWlJvIiwidiI6MSwicCI6IntcInVcIjozMTE2NTM0MCxcInZcIjoxLFwidXJsXCI6XCJodHRwczpcXFwvXFxcL2hlbHAub3BlbmFpLmNvbVxcXC9lblxcXC9cIixcImlkXCI6XCJlZTZlMjhkNDRlOGQ0Yzk3YWRkMTliMTg1NmM1OTAzY1wiLFwidXJsX2lkc1wiOltcIjc3NTAwMDcwYzhjNTAzZTc2YTk4MTdmMGRhZjFiNTQ1ZGIyNTE1YThcIl19In0). We will review appeals within one business day and will contact you if we reinstate access to the API.

Best,
The OpenAI team

**Comment by szh-bash** [comment]
你 OpenAI 号是咋来的啊

**Reply by Panlichen** [reply]
微软账号，注册的，手机验证用的网上买的虚拟号

**Comment by ZekkoRyu** [comment]
OpenAI最近更新了策略，如果使用中国大陆/香港节点访问API是有可能被封禁的。
OpenAI Usage policies：[https://openai.com/policies/usage-policies](https://openai.com/policies/usage-policies)

**Reply by Panlichen** [reply]
我用的是印度的代理，肉身在大陆

**Reply by ZekkoRyu** [reply]
> 我用的是印度的代理，肉身在大陆

代理是开的全局么

**Reply by Panlichen** [reply]
是的

**Reply by ZekkoRyu** [reply]
> 是的

那就不太清楚了，我也刚被封，都不太敢用了。。。

**Reply by missy0u** [reply]
这个划词用的频率极低，今天我也被封了。。。

**Reply by Panlichen** [reply]
搞了个新号，不太敢再用了。。
